packages/strapi-design-system/src/v2/Breadcrumbs/CrumbSimpleMenu.tsx

```diff
@@ -16,15 +15,15 @@ const StyledButton = styled(Button)`
   }
 `;
 
-export const CrumbSimpleMenu = ({ children, ...props }) => (
-  <SimpleMenu noBorder icon={<CarretDown />} as={StyledButton} size="S" {...props}>
+export interface CrumbSimpleMenuProps extends ButtonProps {
+  'aria-label': string;
+  icon?: React.ReactElement;
+}
+
+export const CrumbSimpleMenu = ({ children, ...props }: CrumbSimpleMenuProps) => (
+  <SimpleMenu icon={<CarretDown />} as={StyledButton} size="S" {...props}>
```

Comment:
Where did the `noBorder` prop go? I'm not sure, but I think to remember it is needed, because it behaves, but doesn't look like a simple menu.

Reply:
But the SimpleMenu has not the noBorder prop, here you can see the error I had with the noBorder attribute

Reply:
Maybe the type definitions are wrong? I'm not sure. In any case: the menu looks broken: https://design-system-git-chore-breadcrumbs-ts-strapijs.vercel.app/?path=/story/design-system-components-v2-breadcrumbs--base

<img width="232" alt="Screenshot 2023-05-24 at 12 03 00" src="https://github.com/strapi/design-system/assets/2244375/3ffd88a1-cef0-44e2-8142-f5898b400a9c">

Reply:
@simotae14 did you solve this puzzle?

Reply:
I've also created an issue, because this is already broken on main: https://github.com/strapi/design-system/issues/1090

Reply:
ok then I can take a look at the issue. Maybe in the meantime you can review this PR which is unrelated to the bug